benchmark/solver/solver.cpp

```diff
@@ -42,6 +42,9 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <iostream>
 
 
+#define DISABLE_FORMATS_COMMAND
+#include "benchmark/utils/common.hpp"
+#undef DISABLE_FORMATS_COMMAND
```

Comment:
I don't think you need to disable the formats here. It should be harmless if they are activated right? The option would nothing but that's normal considering we expect the format to be listed in the `json` input.